quickstarts/microsoft.digitaltwins/digitaltwins-with-function-time-series-database-connection/modules/tsdbconnection.bicep

```diff
@@ -0,0 +1,61 @@
+@description('Existing Digital Twin resource name')
+param digitalTwinsName string
+@description('Existing Event Hubs namespace resource name')
+param eventHubsNamespaceName string
+@description('Existing event hub name')
+param eventHubName string
+@description('Existing Azure Data Explorer cluster resource name')
+param adxClusterName string
+@description('Existing database name')
+param databaseName string
+
+@description('Name given to table in database')
+param databaseTableName string
+
+// Gets Digital Twins resource
+resource digitalTwins 'Microsoft.DigitalTwins/digitalTwinsInstances@2022-05-31' existing = {
+  name: digitalTwinsName
+}
+
+// Gets Azure Data Explorer cluster resource
+resource adxCluster 'Microsoft.Kusto/Clusters@2022-02-01' existing = {
+  name: adxClusterName
+}
+
+// Gets Event Hubs namespace resource
+resource eventHubsNamespace 'Microsoft.EventHub/namespaces@2021-11-01' existing = {
+  name: eventHubsNamespaceName
+}
+
+// Gets database resource
+resource database 'Microsoft.Kusto/clusters/databases@2022-02-01' existing = {
+  name: '${adxClusterName}/${databaseName}'
+}
+
+// Assigns Digital Twins admin assignment to database
+resource digitalTwinsToDatabasePrincipalAssignment 'Microsoft.Kusto/clusters/databases/principalAssignments@2022-02-01' = {
+  name: '${adxClusterName}/${databaseName}/${guid(digitalTwins.id, resourceGroup().id, 'Admin')}'
```

Comment:
change the roleAssignment name here as in the previous file (pass in the principalId instead of the resourceName)

Reply:
I have same comment on this as the previous file